hilti/toolchain/src/compiler/visitors/printer.cc

```diff
@@ -799,7 +799,7 @@ struct Visitor : visitor::PreOrder<void, Visitor> {
 
     void operator()(const type::Interval& n) { out << const_(n) << "interval"; }
 
-    void operator()(const type::Member& n) { out << const_(n) << "<member>"; }
+    void operator()(const type::Member& n) { out << const_(n) << n.id(); }
```

Comment:
Not sure this is exactly due to this line, but the printing of the interval operators does not seem to round-trip, e.g.,

```console
$ cat x.hlt
module x {
global i = interval(10);
global j = interval_ns(10);
}
$ hiltic -p x.hlt
module x {

global interval i = 10.000000s;
global interval j = 0.000000s;

}
```

In addition to this not compiling, we also loose the value for `interval_ns` since the default real formatting doesn't have enough precision (probably preexisting).

Reply:
That's elsewhere in the printer, but good point,  I'll tweak it to print out the constructors so that the types match the original ones.

Reply:
It's now printing the types when it's an integer type with width less then 64, that avoids any ambiguity.